recipes/heavydb/meta.yaml

```
@@ -0,0 +1,575 @@
+        - test -f "${PREFIX}/etc/conda/{{ state }}.d/{{ PKG_NAME }}_{{ state }}.sh"
+        {% endfor %}
+        - |
+          source ${PREFIX}/etc/conda/activate.d/{{ PKG_NAME }}_activate.sh
+          test -n "${HEAVYAI_ROOT_PATH+x}"
+          ldd ${PREFIX}/lib/libDBEngine.so
+
+    about:
+      home: https://heavy.ai
+      license: Apache-2.0
+      license_family: APACHE
+      license_file: LICENSE.md
+      summary: The HeavyDB database
+
+      description: |
+        HeavyDB is an in-memory, column store, SQL relational database
+        that was designed from the ground up to run on GPUs.
+      doc_url: https://docs.heavy.ai/
+      dev_url: https://github.com/heavyai/heavydb
+
+  - name: pyheavydbe
+    version: {{ version }}
+    script: build-heavydb.sh
+    build:
+      string: {{ py_build_string_ext }}
+      skip: true  # [cuda_compiler_version not in (undefined, "None", "11.0") or not linux64]
+      skip: true  # temporarily skipping pyheavydbe build
+      missing_dso_whitelist:
+        - '*/libcuda.*'  # [cuda_compiler_version not in (undefined, "None")]
+      track_features:
+        {{ "- arrow-cuda" if cuda_enabled else "" }}
+      ignore_run_exports:
+        - arrow-cpp {{ arrow_version }}
+        - blosc
+        - cudatoolkit    # [cuda_compiler_version not in (undefined, "None")]
+        - double-conversion
+        - fmt
+        - glog
+        - libarchive
+        - libgdal
+        - libllvm{{ llvm_version }}
+        - libpng
+        - librdkafka
+        - libthrift {{ thrift_version }}
+        - pyarrow {{ arrow_version }}
+        - python
+        - tbb
+      run_exports:
+        - {{ pin_subpackage('pyheavydbe',  max_pin='x.x.x') }}
+    requirements:
+      build:
+        # c compiler is specified here to get run constraint pins correct, presumably..
+        - {{ compiler('c') }}
+        - {{ compiler('cgo') }}
+        - {{ compiler('cxx') }}
+        - {{ compiler("cuda") }}  # [cuda_compiler_version not in (undefined, "None")]
+        - clangdev {{ llvm_version }}
+        # clang++ is used for generating the bytecodes of extension functions
+        - clangxx {{ llvm_version }}
+        - cmake
+        - llvmdev {{ llvm_version }}
+        - make
+        - ninja
+        - maven
+      host:
+        - arrow-cpp ={{ arrow_version }}=*{{ build_ext }}
+        - bisonpp
+        - blosc
+        - boost-cpp
+        - clangdev {{ llvm_version }}
+        - cython
+        - double-conversion
+        - flex
+        - fmt
+        - glog
+        - llvmdev {{ llvm_version }}
+        - llvm {{ llvm_version }}
+        - libarchive
+        - libgdal
+        - libpng
+        - librdkafka
+        - numpy
+        - pip
+        - pyarrow ={{ arrow_version }}=*{{ build_ext }}
+        - python
+        - pytest
+        - tbb
+        - tbb-devel
+        - thrift-cpp {{ thrift_version }}
+        - {{ pin_subpackage('heavydbe', exact=True) }}
```

This might be better expressed like this:

```suggestion
        - {{ pin_subpackage('heavydbe', max_pin='x.x.x') }}
```

`exact=True` has some issues with multiple outputs and subpackages, iirc.